### ISSUE TITLE
测试中训练集和测试集选择的问题

### ISSUE THREAD

**Issue by SuperLuckyQiu** [shijxcs/LIFT#1]
For the test at the end of training, why would you use the one loaded from the training set instead of the train_test_loader, instead of the test_loader loaded from the test set, and the accuracy of the test from the training set is not accurate, right?

**Comment by shijxcs** [comment]
Thank you for your concern. The `test()` function defaults to using the `test_loader` since the parameter `mode` is default set to `"test"`.

The `train_test_loader` is only used when adding an option `test_train True`. For instance, if you run `python main.py -d imagenet_lt -m clip_vit_b16_peft test_train True`, you will obtain the training accuracy on ImageNet-LT. This can be used for comparing training/test accuracy, as presented in Table 8 in our paper. Aside from this, all results are calculated from the test set.
